spec/models/bib_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Bib, type: :model do
+  subject(:bib) { build(:bib_with_holdables) }
+
+  it 'will generate holdables when supplied with a body that has a callList' do
+    expect(bib.holdables.count).to be 8
+  end
+
+  it 'will show you an author' do
+    expect(bib.author).to be 'Hill Street blues (Television program)'
+  end
+
+  context 'with an item list of only one item' do
+    it 'will not generate any new holds' do
```

Comment:
@cdmo no expect here?

Reply:
oops left 3 pending (without `xit`). Will add specs and commit.